### ISSUE TITLE
Make refresh interval configurable per Sync

### ISSUE THREAD

**Issue by rguldener** [NangoHQ/nango#26]
Currently the refresh interval is fixed at 60 minutes per sync.
We should add an option to the Sync config to make it configurable per Sync: Some syncs need more frequent refreshes, others less.

To consider:
We probably should not start a new refresh for a Sync before the last one has finished.

**Comment by rguldener** [comment]
Turns out this is a duplicate of #21
